Scripts/aarch64_fit_native.py

```diff
@@ -23,6 +23,8 @@
         tuple([0x51, 0x800]): "cortex-a73", # Kryo 2xx Gold
         tuple([0x51, 0x802]): "cortex-a75", # Kryo 3xx Gold
         tuple([0x51, 0x804]): "cortex-a76", # Kryo 4xx Gold
+        # Apple M1 Parallels hypervisor
+        tuple([0x41, 0x0]): "cortex-a76", # Placeholder CPU that is atleast ARMv8.2
```

Comment:
XXX: This should check the clang version used. If it is clang-12 or newer then `-mcpu=apple-a14` is available. Otherwise should choose A13 as the next best featureset

Reply:
Fixed